docker/gen-run-image

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
+set -e
+
+PATH="$(git config -f .gitmodules submodule.beaver.path)/bin:$PATH"
+ROOT=$(git rev-parse --show-toplevel)
+
+export DIST="xenial"
+export DMD="dmd1"
+export F="production"
+
+cd $ROOT
+BEAVER_DOCKER_Iimg="sociomantictsunami/dhtnode-builder" beaver dlang install
+beaver dlang make pkg
+
+rm -rf docker-tmp
+mkdir docker-tmp
+cp build/$F/pkg/dhtnode-d*.deb docker-tmp/dhtnode.deb
+cp doc/etc/dht.config.ini docker-tmp/config.ini
+
+cat > docker-tmp/Dockerfile <<- file
+FROM ubuntu:xenial
+COPY dhtnode.deb /packages/
+RUN apt update
+RUN mkdir -p /srv/dhtnode/dhtnode-0 && apt install -y /packages/*.deb
+COPY config.ini /srv/dhtnode/dhtnode-0/etc/
```

Comment:
I assume here that whatever config is in `doc/dht.config.ini` is suitable for local single-node deployment as is.

Reply:
Yeah, it should be ok.